plugins/module_utils/network/eos/config/lacp_interfaces/lacp_interfaces.py

```diff
@@ -240,9 +240,13 @@ def _state_deleted(want, have):
 def generate_commands(interface, to_set, to_remove):
     commands = []
     for key in to_remove.keys():
+        if key == "rate":
+            continue
```

Comment:
Is this necessary at all? It's possible I'm misremembering how aliases work, but I thought they should just get saved as the key name and not the alias.

Reply:
As far as I checked, when the alias is used, both the alias and key name are set to the value. In the above case, both `lacp timer` and `lacp rate` gets set.